### PR TITLE
Removes the AI's departmental radio

### PR DESCRIPTION
**Pull request by MrPerson**
Also removes the freerange var on intercoms because it would only serve to keep departmental radio in a more roundabout way.

I'll add a changelog and whatever later. I'm just getting this up now so people can talk about it calmly and rationally before I go to sleep.

### CODE REVIEW

**Comment by WJohn** [comment]
You're going to be fighting a thousand map conflicts with anturk's PR up right now.

**Comment by Sarcalogo** [comment]
Why? Terrible idea. Makes no sense from an IC or OOC perspective.

**Comment by GunHog** [comment]
First he removes NTSL, now he removes this. :-1:  This makes things even worse than before. Have I told you how _boring_ it is to only have two radio channels?

**Comment by Aloraydrel** [comment]
This really does not need to be done.

**Comment by drynwyn** [comment]
Maybe remove Command from the AI, but I would keep the rest.

**Comment by TheNightingale** [comment]
I'm not so sure this is a good idea. The problem isn't with the AI; it's with how people play AI. If they're being used as validhunting tools, address that, don't just remove their departmental radios FNR. Security cyborgs get bright red letters saying "YOU OBEY YOUR LAWS NOT THE LAW", why not give AIs that too?

**Comment by Supermichael777** [comment]
Make it a var and leave it on. this is the kind of change that affects down streams. also ai private is now pointless because you can't tune radios to it any more for hiden mics.

**Comment by MrStonedOne** [comment]
:-1:

Remove command and security and give a button on key card swipers to re-enable the ai getting them. (or better yet, just make red alert give the ai command/security access)

**Comment by RemieRichards** [comment]
:-1: This is needless.

**Comment by Shadowlight213** [comment]
:-1: Make it a config option or something.
Also, #13509

File: _maps/RandomZLevels/snowdin.dmm [hunk 1]
```diff
@@ -583,7 +583,7 @@
 "lo" = (/obj/structure/table,/turf/simulated/floor/plasteel/purple{baseturf = /turf/simulated/floor/plating/asteroid/snow},/area/awaymission/snowdin/sekret)
 "lp" = (/obj/structure/filingcabinet,/turf/simulated/floor/plasteel/white{baseturf = /turf/simulated/floor/plating/asteroid/snow},/area/awaymission/snowdin/sekret)
 "lq" = (/turf/simulated/floor/plasteel/purple{baseturf = /turf/simulated/floor/plating/asteroid/snow},/area/awaymission/snowdin/sekret)
-"lr" = (/obj/structure/table,/obj/item/device/radio/intercom{broadcasting = 0; dir = 8; freerange = 1; listening = 1; name = "Pirate Radio Listening Channel"; pixel_x = 0},/turf/simulated/floor/plasteel/purple{baseturf = /turf/simulated/floor/plating/asteroid/snow},/area/awaymission/snowdin/sekret)
```
Comment (Shadowlight213) [inline _maps/RandomZLevels/snowdin.dmm]: Congrats on nerfing the DJ station.

Comment (duncathan) [inline _maps/RandomZLevels/snowdin.dmm]: Unforgivable :-1:

**Comment by PKPenguin321** [comment]
yeah just remove sec and command, as long as we still have the AI it should at least be able to communicate with certain departments (but command/sec is kinda validhunty)

**Pull request closed without merging**